src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -11,6 +11,11 @@
 #include "alltoall/alltoall.h"
 #include "core/ucc_team.h"
 #include <sys/shm.h>
+<<<<<<< HEAD
```

Comment:
this is some merge bug, please check there are no compilation issues

Reply:
@Sergei-Lebedev yeah, I saw that there is merge conflict, please check now. it should be resolved